.github/workflows/release.yml

```diff
@@ -58,30 +57,24 @@ jobs:
         shell: bash
 
       - name: Install dependencies
-        working-directory: ./vscode
         run: npm ci
 
-      # Run tests on Linux
+      # Run tests on different OSes
       - name: Run tests (Linux)
         if: matrix.arch == 'linux'
         working-directory: ./vscode
         run: xvfb-run -a npm run test
 
-      # Run tests on macOS
       - name: Run tests (macOS)
         if: matrix.arch == 'macos'
-        working-directory: ./vscode
         run: npm test
 
-      # Run tests on Windows
       - name: Run tests (Windows)
         if: matrix.arch == 'windows'
-        working-directory: ./vscode
         shell: cmd
         run: npm test
 
       - name: Build Package
-        working-directory: ./vscode
         run: npm run package
 
       - name: Generate .vsix package
```

Comment:
another case where we leave the `./vscode` folder - if it's a special case please leave a comment

Reply:
This is a repeat from the `ci-repo.yml`.  I don't mind a comment not being here mostly because the packaging script is due some updating.

Reply:
OK. I just got confused that working folder is left only in some places.